core.spec.md

```diff
@@ -249,6 +250,36 @@ directive @eg(data: eg__Data) on FIELD_DEFINITION
 directive @core(feature: String!, as: String) repeatable on SCHEMA
 ```
 
+###! for: [core__Purpose!]
+
+An optional list of [purposes](#core__Purpose) for this feature. This provides a hint to processors and data cores as to whether they can safely ignore a given feature.
+
+By default, core features SHOULD fail open. This means that an unknown feature SHOULD NOT prevent a schema from being served or processed. Instead, consumers SHOULD ignore unknown features and serve or process the rest of the schema normally.
```

Comment:
What does 'fail open' mean? I interpreted this as 'failing to open', meaning unknown features SHOULD prevent a schema from being served or processed, but the next sentence states exactly the opposite (which makes sense to me, just wondering whether 'fail open' includes any additional expected behavior).

Reply:
“fail open” is a common term describing a system which keeps operating even in the presence of errors or unknown conditions. the metaphor is that of a valve: valves may fail open, continuing to flow in a failed state, or they may fail safe, stopping the flow if a problem has occurred.

i would say we should clarify this, but we do in the next sentence, and it seems like a google search would satisfy any remaining questions:

> A system set to fail open does not shut down when failure conditions are present. Instead, the system remains “open” and operations continue as if the system were not even in place. This strategy is used when access is deemed more important that authentication.

(google one box 👆🏾)

Reply:
Ah, I knew 'fail safe' but never realized the metaphor it was based on! I don't know how many others would be unfamiliar with the expression, but I would say asking people to google this is no alternative to clarifying the meaning in the document (or avoiding the expression altogether).